src/main/java/frc5124/robot2020/commands/shooter/ShootVelocity.java

```diff
@@ -0,0 +1,29 @@
+/*----------------------------------------------------------------------------*/
```

Comment:
not sure you're using the instantCommand class as intended. My understanding is that you can bind a button to a command inline using instant commands which means you wouldn't need the extra verbosity of an explicit command file and this is intended for advanced developers. But I expressed my opinion before that using explicit command files is better for readability for all of the novice developers on our team. I thought we agreed to standardize on CommandBase.

Reply:
adjusted